src/basics/Problem41.java

```diff
@@ -11,6 +11,6 @@ public static void main(String []args) {
     
     char character = letter;
     int ascii = (int) character;
```

Comment:
Why do you need `character` at all, why can't you just do:
```
int ascii = (int) letter;
```

Reply:
Is it okey now?